Makefile

```diff
@@ -23,3 +23,26 @@ docker-build:
 .PHONY: offsets
 offsets:
 	cd offsets-tracker; OFFSETS_OUTPUT_FILE="../pkg/inject/offset_results.json" go run main.go
+
+.PHONY: install-tools
+install-tools:
+	go install github.com/google/go-licenses@latest
```

Comment:
Can we use [the standard `go.mod` + `tools.go` pattern](https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module) to pin the tooling dependencies.

You can check how it is done in other Go-based OTel repositories:
- https://github.com/open-telemetry/opentelemetry-go-build-tools
- https://github.com/open-telemetry/opentelemetry-go
- https://github.com/open-telemetry/opentelemetry-collector

We should also add an entry in `dependabot.yml` to have automated PRs for bumping the dependencies.

Reply:
I did noticed this repo wasn't doing this yet.  As this is the first tool I can add these building blocks as part of this PR.

Reply:
Added